jsonschema-core.xml

```diff
@@ -313,43 +313,70 @@
                 <section title="JSON Schema Objects and Keywords">
                     <t>
-                        Object properties that are applied to the instance are called keywords,
-                        or schema keywords.  Broadly speaking, keywords fall into one
-                        of five categories:
+                        Object properties that are applied to the instance are called keywords, or schema keywords.
+                        Two properties in the same schema object are referred to as adjacent keywords.
+                        As an authoring convenience, adjacent keywords can sometimes affect each other in a way that a property from a different schema (including sub-schemas) cannot.
+                    </t>
+                    <t>
+                        Keywords are classified into one or more of six categories:
                     </t>
                     <dl>
-                        <dt>identifiers</dt>
+                        <dt>Core keywords</dt>
+                        <dd>
+                            Keywords that may configure how the schema is processed,
+                            and can modify the meaning of other keywords in the schema.
+                        </dd>
+                        <dt>Assertion keywords</dt>
                         <dd>
-                            control schema identification through setting a IRI
-                            for the schema and/or changing how the base IRI is determined
+                            Keywords that may decide an instance is invalid.
                         </dd>
-                        <dt>assertions</dt>
+                        <dt>Annotation keywords</dt>
                         <dd>
-                            produce a boolean result when applied to an instance
+                            Keywords that produce output generated from an instance.
                         </dd>
-                        <dt>annotations</dt>
+                        <dt>Applicator keywords</dt>
                         <dd>
-                            attach information to an instance for application use
+                            A keyword that has one or more subschemas,
+                            that are applied against the instance,
+                            or some value within the instance,
+                            and logically aggregated together.
                         </dd>
-                        <dt>applicators</dt>
+                        <dt>Reserved keywords</dt>
                         <dd>
-                            apply one or more subschemas to a particular location
-                            in the instance, and combine or modify their results
+                            Keywords that are permitted in a schema,
+                            but that do not do anything by themselves,
+                            though their value may be read by or referred to by another keyword.
```

Comment:
Do you have an example of a "reserved" keyword?  What would the purpose of this be?